client/components/ReviewListEntry.jsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/require-default-props */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ReviewListEntry = ({
+  profilePic,
+  name,
+  date,
+  rating,
+  review,
+  attachedPic,
+  purchasedItemName,
+  purchasedItemPic,
+}) => (
```

Comment:
That's a really clever way of deconstructing your props.  I hadn't considered doing that!